aws-api/src/androidTest/assets/create-comment.graphql

```diff
@@ -1,5 +1,5 @@
-mutation CommentOnEvent($eventId: ID!, $content: String!, $createdAt: String!) {
-  commentOnEvent(eventId: $eventId, content: $content, createdAt: $createdAt) {
+mutation CommentOnEvent($eventId: ID!, $commentId:String!, $content: String!, $createdAt: String!) {
```

Comment:
I think the request will work either way, but the name of the mutation should probably be changed to `CreateEvent` as well.

```suggestion
mutation CreateEvent($eventId: ID!, $commentId:String!, $content: String!, $createdAt: String!) {
```